cpp/benchmarks/io/parquet/parquet_reader_benchmark.cpp

```diff
@@ -18,36 +18,37 @@
 
 #include <benchmarks/common/generate_benchmark_input.hpp>
 #include <benchmarks/fixture/benchmark_fixture.hpp>
-#include <benchmarks/io/cuio_benchmarks_common.hpp>
 #include <benchmarks/synchronization/synchronization.hpp>
 
 #include <cudf/io/parquet.hpp>
 
 // to enable, run cmake with -DBUILD_BENCHMARKS=ON
 
-constexpr int64_t data_size = 512 << 20;  // 512 MB
+constexpr size_t data_size         = 512 << 20;
```

Comment:
Is this intended to be different from the 256 << 20 values used for orc and csv?

Reply:
Basically these are chosen to be as large as possible without running into OOM issues on most GPUs. And JSON and CSV have higher memory overhead, hence the lower size.
I just tried using 256MB in ORC and Parquet benchmarks, and I'm getting lower throughput by up to 10%. I'd prefer the size to be on the performance plateau.

Reply:
👍